docs/qcfractal/source/managers_faq.rst

```diff
@@ -0,0 +1,130 @@
+        link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+        inet 127.0.0.1/8 scope host lo
+    3: eno49.4010@eno49: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP qlen 1000
+        inet 10.XX.Y.Z/16 brd 10.XX.255.255 scope global eno49.4010
+    4: eno49.4049@eno49: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP qlen 1000
+        inet 198.XX.YYY.ZZZ/24 brd 198.XX.252.255 scope global eno49.4049
+
+and a bunch of other lines in between I have removed, the lines I have left with the ``XX``, ``YYY`` and ``ZZZ`` will
+have actual numbers there.
+There are a few important things: the headers (``lo``, ``eno.49...``, yours will be different) and the addresses where
+the ``XX`` placeholders are. First thing: Ignore the ``lo`` adapter, every machine should have one. On the head node,
+you will probably see an adapter who has a ``inet`` that looks like a normal IP address, and probably one which
+looks like it has a ``10.something`` IP address. The compute node will probably have an adapter which is
+only the ``10.something``. These ``10.something`` IP addresses are often intranet communication only, so the compute
+nodes cannot reach the broader internet. What will be different is name of the ethernet adapter housing
+the ``10.something`` will be *different* on the head node and the compute node.
+
+In this case, in your YAML file, add a line in ``dask`` called ``interface`` and set it to the name of the
+adapter which is shared. So for this it would be:
+
+.. code-block:: yaml
+
+    dask:
+     interface: "eno49.4049"
+
+plus all the rest of your YAML file. You can safely ignore the bit after the ``@`` sign.
+
+If there isn't a shared adapter name, try this instead:
+
+.. code-block:: yaml
+
+    dask:
+     ip: "10.XX.Y.Z"
+
+And replace the ``.XX.Y.Z`` bit with the code which has the intranet IP of the *head node*. This option
+acts as a pass through to the Dask :term:`Worker` call and tells the worker to try and connect to the
+head node at that IP address. If that still doesn't work, contact us.
+
+
+Other variants:
+
+- "My jobs start and stop instantly"
+- "My jobs restart forever"
+
+
+My Conda Environments are not Activating
+++++++++++++++++++++++++++++++++++++++++
+
+You likely have to ``source`` the Conda ``profile.d`` again first. See also
+`<https://github.com/conda/conda/issues/8072>`_
+
+
+My jobs appear to be running, but only one (or few) workers are starting
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+If the jobs appear to be running (and the Manager is reporting they return successfully),
+a few things may be happening.
+
+- If jobs are completing very fast, the :term:`Adapter` may not feel like it needs to start more
+  :term:`workers<Worker>`, which is fine.
+- Check your ``manger.max_tasks`` arg to pull more :term:`tasks<Task>` from the :term:`Server` to fill
+  the jobs you have started.
```

Comment:
A general organization strategy to consider as the docs expand is the FAQ page https://raw.githubusercontent.com/psi4/psi4/master/doc/sphinxman/source/build_faq.rst and http://psicode.org/psi4manual/master/build_faq.html . Let's you accumulate content separately from the reasons why ppl would to meet that content.